config/manifests/bases/kustomization.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- lvm-operator.clusterserviceversion.yaml
```

Comment:
The name of this file can change (downstream build uses odf-lvm-operator.clusterserviceversion.yaml) depending on the value of $BUNDLE_PACKAGE (see rename-csv)

Reply:
Thanks for pointing it out, made changes accordingly.